test/promise.spec.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-env mocha */
```

Comment:
This is super important :P no I just forgot to delete it before comitting, fell free to remove